service-deployment/bootstrap/rest-server/rest-server.yaml.template

```diff
@@ -53,4 +57,8 @@ spec:
           containerPort: 8080
           hostPort: {{ clusterinfo['restserverinfo']['server_port'] }}
       imagePullSecrets:
-      - name: {{ clusterinfo['dockerregistryinfo']['secretname'] }}
\ No newline at end of file
+      - name: {{ clusterinfo['dockerregistryinfo']['secretname'] }}
+      volumes:
+      - name: database-path
+        hostPath:
+          path: /var/databases
```

Comment:
>/var/databases [](start = 16, length = 14)

make this configurable in clusterconfig-example.yaml

Reply:
I would suggest to use a common parent folder to store the all data on host for PAI, for example, "/var/pai/webportal" so that we store the all PAI data under "/var/pai".

Reply:
We can't strict the data on one host, so `/var/pai` path on each host?
For auth database, only need to store on the host where runs the rest-server pod.